.githooks/pre-commit

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+#
```

Comment:
shouldn't all git hook scripts go under .git/hooks ?

Reply:
after installing the git hook, you'll find it there. But `.git` is not saved in the git history itself, so it needs to be outside if you want to share it